.idea/dictionaries/talesra.xml

```diff
@@ -0,0 +1,8 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="talesra">
+    <words>
+      <w>uncategorized</w>
+      <w>xssi</w>
+    </words>
+  </dictionary>
+</component>
```

Comment:
Please add newline before EOF, here and elsewhere.

You can check the "Files Changed" tab to see which files need it.

Reply:
@seanlip I have added new line at the end in the files which I have written and some files are automatically generated in which EOF is not there. So should I change those generated files too? Like the one which you have mentioned.

Reply:
Can you find a way to generate them so that a newline is added at EOF?

Reply:
(In any case, the short answer to your question is yes. Though it may be worth checking with @BenHenning whether we should be checking in generated files into source control.)

Reply:
@seanlip I have activated this from settings in android studio.

Reply:
I don't understand, sorry. What have you activated?

Also, please check your spelling in .gitignore. That might be source of some of the issues.

Reply:
> Can you find a way to generate them so that a newline is added at EOF?

In Android Studio -> Settings -> Editor Settings we have an option to select "Add newline before EOF" in generated files. That way even the generated files will have a new line before EOF.

Reply:
This has been solved.